source/upgrade/v3.txt

```diff
@@ -61,6 +61,11 @@ Version 3.0 Breaking Changes
   and {+mdb-server+} versions, visit the
   :ref:`Compatibility <csharp-compatibility-tables>` page.
 
+- The classes, methods, and properties ``MongoDB.Driver.Core`` namespace that were
```

Comment:
Should this be "classes, methods, and properties _in the_ ``MongoDB.Driver.Core`` namespace"?

Reply:
yep! good catch.